azure-pipelines.yml

```diff
@@ -23,6 +23,8 @@ jobs:
       architecture: x64
       pool: Hosted VS2017
       queue: Windows.Amd64.ClientRS4.DevEx.15.8.Perf
+      csproj: src\benchmarks\micro\MicroBenchmarks.csproj
```

Comment:
You don't seem to have added the kind parameter here.

Reply:
For the micro benchmarks I have added it some time ago, but GH does not show it here because it was not changed in this PR.

When you click "View file" you should see it. https://github.com/dotnet/performance/blob/e1cce9b2336c38e2dd3bdd396707452a4d57eb56/azure-pipelines.yml#L22

Reply:
Ah. I see.

Reply:
I didn't notice that the only thing changed in the line in the template was to add the ml kind as a supported option.